extra/renderer-and-libwebrtc-tests.js

```diff
@@ -442,15 +442,10 @@ function TestMediaRenderCatpure(videoEl) {
 	});
 }
 
-var testMediaRenderCatpureAnimateFrame;
-function TestMediaRenderCatpureAnimate() {
-	TestMediaRenderCatpure(peerVideoEl);
-	testMediaRenderCatpureAnimateFrame = requestAnimationFrame(TestMediaRenderCatpureAnimate);
-}
-
```

Comment:
This doesn't appear to be used anywhere.  Let me know if that is incorrect.

Reply:
This may be use in the console to be tested

Reply:
Sounds good, I'll add it back.  There is a typo `Catpure`.  Want that fixed?

Reply:
Sure

Reply:
Done